lxd/containers.go

```diff
@@ -1719,6 +1737,16 @@ func (c *lxdContainer) setupMacAddresses(d *Daemon) error {
 			}
 		}
 
+		/*
+		 * propagate the name of the device down for nics if the user
+		 * didn't specify one. This is probably what they wanted
+		 * anyway, and lets us match against network names once they go
+		 * into lxc, e.g. to pull out the veth name for `lxc info`.
+		 */
+		if _, ok := d["name"]; !ok {
+			d["name"] = name
+		}
+
```

Comment:
Just to confirm, "name" there refers to the device name INSIDE the container, correct?

If so, then that's fine, if it instead refers to the device name OUTSIDE the container, then we shouldn't do this.